.formatter.exs

```diff
@@ -1,4 +1,4 @@
 # Used by "mix format"
 [
-  inputs: ["{mix,.formatter}.exs", "{config,lib,test}/**/*.{ex,exs}"]
+  inputs: ["{mix,.formatter}.exs", "{lib,test}/**/*.{ex,exs}"]
```

Comment:
`mix new app` doesn't create `config` dir, so I think it's now more idiomatic to not have `config` dirs in libraries if they are not necessary. And with some other changes in this PR, `config` dir is not longer necessary :)

Reply:
agreed. I wanted to get rid of it and use a supervisor instead as this is the better way. I inherited earlier this year the lib and still trying to find time to go fix it but seems that you are going in the direction I was planning